src/test/regress/expected/upgrade_columnar_before.out

```diff
@@ -0,0 +1,143 @@
+SELECT :relfilenode_post_vacuum_full <> :relfilenode_pre_vacuum_full AS relfilenode_changed;
+ relfilenode_changed
+---------------------------------------------------------------------
+ t
+(1 row)
+
+INSERT INTO test_vacuum_full SELECT * FROM generate_series(11, 13);
+SELECT count(*) FROM test_vacuum_full;
+ count
+---------------------------------------------------------------------
+    13
+(1 row)
+
+-- alter column type
+CREATE TABLE test_alter_type (a int) USING columnar;
+INSERT INTO test_alter_type SELECT * FROM generate_series(1, 3);
+SELECT count(*) FROM test_alter_type;
+ count
+---------------------------------------------------------------------
+     3
+(1 row)
+
+SELECT relfilenode AS relfilenode_pre_alter
+FROM pg_class WHERE oid = 'test_alter_type'::regclass::oid \gset
+ALTER TABLE test_alter_type ALTER COLUMN a TYPE text;
+SELECT relfilenode AS relfilenode_post_alter
+FROM pg_class WHERE oid = 'test_alter_type'::regclass::oid \gset
+SELECT :relfilenode_pre_alter <> :relfilenode_post_alter AS relfilenode_changed;
+ relfilenode_changed
+---------------------------------------------------------------------
+ t
+(1 row)
+
+INSERT INTO test_alter_type SELECT * FROM generate_series(17, 19);
+SELECT count(*) FROM test_alter_type;
+ count
+---------------------------------------------------------------------
+     6
+(1 row)
+
+-- materialized view
+CREATE MATERIALIZED VIEW matview(a, b) USING columnar AS
+SELECT floor(a/3), array_agg(b) FROM test_retains_data GROUP BY 1;
+SELECT relfilenode AS relfilenode_pre_refresh
+FROM pg_class WHERE oid = 'matview'::regclass::oid \gset
+REFRESH MATERIALIZED VIEW matview;
+SELECT relfilenode AS relfilenode_post_refresh
+FROM pg_class WHERE oid = 'matview'::regclass::oid \gset
+SELECT :relfilenode_pre_alter <> :relfilenode_post_alter AS relfilenode_changed;
+ relfilenode_changed
+---------------------------------------------------------------------
+ t
+(1 row)
+
+--
+-- Test that we retain options
+--
+SET cstore.stripe_row_count TO 5000;
+SET cstore.block_row_count TO 1000;
+SET cstore.compression TO 'pglz';
+CREATE TABLE test_options_1(a int, b int) USING columnar;
+INSERT INTO test_options_1 SELECT i, floor(i/1000) FROM generate_series(1, 10000) i;
+CREATE TABLE test_options_2(a int, b int) USING columnar;
+INSERT INTO test_options_2 SELECT i, floor(i/1000) FROM generate_series(1, 10000) i;
+SELECT alter_columnar_table_set('test_options_2', block_row_count => 2000);
+ alter_columnar_table_set
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT alter_columnar_table_set('test_options_2', stripe_row_count => 6000);
+ alter_columnar_table_set
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT alter_columnar_table_set('test_options_2', compression => 'none');
+ alter_columnar_table_set
+---------------------------------------------------------------------
+
+(1 row)
```

Comment:
nit; if you want you can set all 3 at once:
```sql
SELECT alter_columnar_table_set('test_options_2', block_row_count => 2000, stripe_row_count => 6000, compression => 'none');
```

Don't think there would be a difference in retaining when set at once, or when updated in separate steps. Unless I am missing something.